Classes/Issues/Labeled/IssueLabeledCell.swift

```diff
@@ -9,95 +9,78 @@
+            NSAttributedStringKey.foregroundColor: Styles.Colors.Gray.medium.color,
+            NSAttributedStringKey.font: Styles.Fonts.secondary
+        ]
+        
         let actionString: String
+        
         switch model.type {
-        case .added: actionString = NSLocalizedString(" added", comment: "")
-        case .removed: actionString = NSLocalizedString(" removed", comment: "")
+        case .added: actionString = NSLocalizedString(" added  ", comment: "")
+        case .removed: actionString = NSLocalizedString(" removed  ", comment: "")
         }
-        let actionAttributes = [
+        
+        attributedString.append(NSAttributedString(string: actionString, attributes: actionAttributes))
+        
+        // Label
+        
+        let labelColor = model.color.color
+        
+        let labelAttributes: [NSAttributedStringKey: Any] = [
+            NSAttributedStringKey.font: Styles.Fonts.secondaryBold,
+            NSAttributedStringKey.backgroundColor: labelColor,
+            NSAttributedStringKey.foregroundColor: labelColor.textOverlayColor ?? .black,
+            MarkdownAttribute.label: model.title
+        ]
+        
+        attributedString.append(NSAttributedString(string: model.title, attributes: labelAttributes))
+        
+        // Date
+        
+        let dateAttributes: [NSAttributedStringKey: Any] = [
+            NSAttributedStringKey.font: Styles.Fonts.secondary,
             NSAttributedStringKey.foregroundColor: Styles.Colors.Gray.medium.color,
-            NSAttributedStringKey.font: Styles.Fonts.secondary
+            MarkdownAttribute.details: DateDetailsFormatter().string(from: model.date)
         ]
-        let action = NSAttributedString(string: actionString, attributes: actionAttributes)
-        let descriptionText = NSMutableAttributedString(attributedString: actor)
-        descriptionText.append(action)
-        descriptionButton.setAttributedTitle(descriptionText, for: .normal)
-
-        let color = UIColor.fromHex(model.color)
-        labelButton.backgroundColor = color
-
-        labelButton.setTitle(model.title, for: .normal)
-        labelButton.setTitleColor(color.textOverlayColor, for: .normal)
-
-        dateLabel.setText(date: model.date)
+        
+        attributedString.append(NSAttributedString(string: "  \(model.date.agoString)", attributes: dateAttributes))
+        
+        // Set
+        
+        let centerVertical = (Styles.Sizes.labelEventHeight - Styles.Fonts.secondaryBold.lineHeight) / 2
+        let inset = UIEdgeInsets(
+            top: centerVertical,
+            left: Styles.Sizes.eventGutter,
+            bottom: centerVertical,
+            right: Styles.Sizes.eventGutter
+        )
+        
+        set(attributedText: NSAttributedStringSizing(
+            containerWidth: 0,
+            attributedText: attributedString,
+            inset: inset,
+            backgroundColor: Styles.Colors.Gray.lighter.color
+        ))
```

Comment:
Eeep, creating these is going to be very expensive. Check out the [source](https://github.com/rnystrom/GitHawk/blob/master/Classes/Systems/NSAttributedStringSizing.swift) of `NSAttributedStringSizing`, it'll setup all TextKit objects and find the size. Plus this will happen _every time_ `configure(...)` is called 😱 (and the width here is `0`, so the calc'd size wont even be used).

Can we move this into `IssueLabeledModel` so its only created w/ the model? See #1068 as a reference.